docs/tutorials/notes-mvvm/index.yml

```diff
@@ -0,0 +1,62 @@
+### YamlMime:Tutorial
+title: "Upgrade your app with MVVM concepts"
+metadata:
+  title: "Tutorial: Use the MVVM toolkit to update the Notes app"
+  description: Upgrade the Notes app from the previous tutorial with the MVVM Community Toolkit.
+  audience: Developer
+  level: Beginner
+  ms.topic: interactive-tutorial
+  ms.date: 12/21/2022
+  author: adegeo
+  ms.author: adegeo
+items:
+- durationInMinutes: 1
+  content: |
+    [!INCLUDE [](includes/intro.md)]
+
+- title: Upgrade the app from .NET 6 to .NET 7
+  durationInMinutes: 3
+  content: |
+    [!INCLUDE [](includes/upgrade.md)]
+
+- title: Brief overview of MVVM
```

Comment:
"Brief overview of MVVM" > "Understand MVVM" for alignment with the other page titles?